src/chef-mover/rebar.config

```diff
@@ -65,7 +65,8 @@
     {test, [
         {deps, [
             {hoax, ".*",
-                {git, "git://github.com/xenolinguist/hoax.git", {tag, "0.9.7"}}},
+                %{git, "git://github.com/xenolinguist/hoax.git", {tag, "0.9.7"}}},
+                {git, "git://github.com/lbakerchef/hoax.git", {branch, "lbaker/fixes-for-erlang20"}}},
```

Comment:
Can we use the latest hoax? Or create an issue to update hoax to the latest.

Reply:
See my comments on hoax here: https://app.zenhub.com/workspaces/chef-infra-5d1b845f1659a92b9fa6194c/issues/chef/chef-server/1796

If you still want to upgrade it, I will be happy to create an issue.